web/packages/teleterm/src/ui/ConnectMyComputer/permissions.ts

```diff
@@ -45,7 +45,6 @@ export function canUseConnectMyComputer(
   return (
     isUnix &&
     rootCluster.loggedInUser?.acl?.tokens.create &&
-    rootCluster.features?.isUsageBasedBilling &&
```

Comment:
Technically, this field is no longer used and could be removed from the protos. However:

1) Fetching it [doesn't cost us anything extra](https://github.com/gravitational/teleport/blob/15dac54519a3eb1209a46e85ccd248f64c7906c7/lib/teleterm/apiserver/handler/handler_clusters.go#L112-L113).
2) Because of the linter for breaking proto changes, we'd have to mark this field as reserved, not being able to reuse it in the future…
3) …but we might actually need it in the future.